syncano_cli/main.py

```diff
@@ -1,38 +1,30 @@
 # -*- coding: utf-8 -*-
-from __future__ import print_function, unicode_literals
+from __future__ import print_function
 
 import sys
-from ConfigParser import ConfigParser, NoOptionError
 
-from syncano_cli import LOG
-from syncano_cli.commands import Configure, Login, Parse, Pull, Push  # noqa
-from syncano_cli.commands_base import parse_arguments
-from syncano_cli.sync.project import Project
+import click
+from syncano.exceptions import SyncanoException
+from syncano_cli.commands import top_level
+from syncano_cli.parse_to_syncano.commands import top_migrate
+from syncano_cli.sync.commands import top_sync
 
 ACCOUNT_KEY = ''
 
-ACCOUNT_CONFIG = ConfigParser()
-
-
-def cli(args):
-    namespace = parse_arguments(args)
-    namespace.project = Project.from_config(namespace.file)
-
-    read = ACCOUNT_CONFIG.read(namespace.config)
-    if read and not namespace.key:
-        try:
-            namespace.key = ACCOUNT_CONFIG.get('DEFAULT', 'key')
-        except NoOptionError:
-            LOG.error('Do a login first.')
-
-    try:
-        namespace.func(namespace)
-    except ValueError as e:
-        LOG.error(e.message)
+cli = click.CommandCollection(
+    sources=[
+        top_level,
```

Comment:
why do you need top_level, top_sync and top_migrate when you already define login, sync and migrate groups ?